src/directives/map.js

```diff
@@ -18,12 +19,14 @@ goog.require('ol.Map');
  * [../examples/simple.html](../examples/simple.html)
  *
  * @htmlAttribute {ol.Map} ngeo-map The map.
+ * @param {ngeo.SyncDataSourcesMap} ngeoSyncDataSourcesMap Ngeo sync
+ *     data sources map service.
  * @return {angular.Directive} Directive Definition Object.
  * @ngInject
  * @ngdoc directive
  * @ngname ngeoMap
  */
-ngeo.mapDirective = function() {
+ngeo.mapDirective = function(ngeoSyncDataSourcesMap) {
```

Comment:
I'm not a big fan of having `ngeo.SyncDataSourcesMap` here. The map initialization should be moved elsewhere (like the map initialization of `ngeo.FeatureOverlayMgr`)